### PR TITLE
ref(storybook): Update webpack configuration

### PR DESCRIPTION
**Pull request by billyvg**
This updates the storybook webpack configuration so that it is more webpack5 ready.

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -6,3 +6,4 @@
 /scripts/
 !.github
 !.github/workflows/scripts/*
+!.storybook
```
Comment (billyvg) [inline .eslintignore:9]: I think by default hidden dirs are ignored by eslint

File: .storybook/webpack.config.js [hunk 2]
```diff
@@ -45,46 +46,34 @@ module.exports = ({config} = {config: emptyConfig}) => {
...
         },
-        {
-          test: /\.(woff|woff2|ttf|eot|svg|png|gif|ico|jpg)($|\?)/,
-          exclude: /app\/icons\/.*\.svg$/,
-          loader: 'file-loader?name=' + '[name].[ext]',
-        },
         {
           test: /\.po$/,
-          loader: 'po-catalog-loader',
-          query: {
```
Comment (billyvg) [inline .storybook/webpack.config.js:85]: `query` is removed in v5

File: .storybook/webpack.config.js [hunk 3]
```diff
@@ -45,46 +46,34 @@ module.exports = ({config} = {config: emptyConfig}) => {
...
+                esModule: false,
+                name: '[name].[hash:6].[ext]',
+              },
             },
           ],
         },
-        {
-          test: /\.(woff|woff2|ttf|eot|svg|png|gif|ico|jpg)($|\?)/,
-          exclude: /app\/icons\/.*\.svg$/,
-          loader: 'file-loader?name=' + '[name].[ext]',
```
Comment (billyvg) [inline .storybook/webpack.config.js:80]: I think this syntax is removed in 5? Not sure but updating to be consistent

File: .storybook/webpack.config.js [hunk 4]
```diff
@@ -45,46 +46,34 @@ module.exports = ({config} = {config: emptyConfig}) => {
...
-          test: /app\/icons\/.*\.svg$/,
-          use: [
-            {
-              loader: 'svg-sprite-loader',
-            },
-            {
-              loader: 'svgo-loader',
-            },
-          ],
-        },
```
Comment (billyvg) [inline .storybook/webpack.config.js:58]: Removed in https://github.com/getsentry/sentry/pull/24582

**Review by matejminar — APPROVED** [review]

**Pull request merged**